pkg/internal/cluster/create/create.go

```diff
@@ -177,41 +177,3 @@ func collectOptions(options ...create.ClusterOption) (*createtypes.ClusterOption
 
 	return opts, nil
 }
-
-func printUsage(name string) {
```

Comment:
are we missing this message?

Reply:
not really, though we did move it off of writing via fmt.Printf which was bad.

I'm going to add a small tweak to it to be closer to before. see above in the code

Reply:
I added a more detailed output, including --kubeconfig if an explicit --kubeconfig was supplied. see above.